package.json

```diff
@@ -65,10 +66,15 @@
     "tslib": "^2.4.0",
     "undici": "^6.12.0"
   },
+  "peerDependencies": {
+    "@opentelemetry/api": "1.x",
+    "@opentelemetry/sdk-trace-base": "1.x"
```

Comment:
This should be in "devDependencies". AFAICT it is only used in the tests. Libraries with OTel instrumentation should, by design, only need a dep on `@opentelemetry/api` (and I suppose potentially on `@opentelemetry/semantic-conventions`).

Reply:
Fixed in ec782d636482749df8ec2386aa399869f7a6569a.